src/platform/arm-nuttx/iotjs_module_gpio-arm-nuttx.cpp

```diff
@@ -48,6 +126,8 @@ Gpio* Gpio::Create(JObject& jgpio) {
 
 GpioArmNuttx::GpioArmNuttx(JObject& jgpio)
     : Gpio(jgpio) {
+  _fd = NULL;
+  _initialized = false;
```

Comment:
you should initialize these like Gpio above:

    : Gpio(jgpio)
    , _fd(NULL)
    ...